build.sbt

```diff
@@ -40,6 +41,16 @@ lazy val fs2 = (projectMatrix in file("streaming/fs2"))
   )
   .dependsOn(core % "compile->compile;test->test")
 
+lazy val zio = (projectMatrix in file("streaming/zio"))
+  .jvmPlatform(
+    scalaVersions = scala2 ++ scala3
+  )
+  .settings(commonSettings)
+  .settings(
+    libraryDependencies += Libraries.sttpClientZio
+  )
+  .dependsOn(core % "compile->compile;test->test")
```

Comment:
** question ** What do we need `test->test` for? I suppose it's needed for
```scala
import sttp.client4.testing.RawStream
```
?

Reply:
It allows you to depend on test dependencies from core module. In our case we use this to access ScalaTest library.

Reply:
`test->test` also means that `src/test` stuff from `core` can be used in `tests` of this module. In this case I checked and it's indeed needed for common test stuff like `sttp.openai.fixtures.*`

Reply:
Right, I forgot to mention it 😅